routes.js

```diff
@@ -6,6 +6,7 @@ var memCache = require('memory-cache');
 module.exports = function (app) {
 
   app.get('/api/channels/:channelId/messages', channels.getPrivateChannelMessage);
+  app.get('/api/channels/:channelId/messagescount', channels.getPrivateChannelMessageCount);
```

Comment:
about the rule of naming in restful, you can refer to 

[http://www.ruanyifeng.com/blog/2014/05/restful_api.html](http://www.ruanyifeng.com/blog/2014/05/restful_api.html)

It does not mean your api is wrong, just follow the specification.